app/styles/_library/_base.forms.scss

```diff
@@ -54,10 +54,12 @@ input[type=checkbox] {
   cursor: pointer;
   display: block;
   float: left;
-  border: 2px solid rgb(var(--color-border-bold));
+  border: 2px solid rgb(var(--color-gray), 0.3);
```

Comment:
colors question: i'm replacing a color token with a specific color here. is there a color token i should use instead, or do we need to update `--color-border-bold`?

Reply:
we'll make a note of this for when we finish up the colors